dallinger/recruiters.py

```diff
@@ -1,11 +1,15 @@
 """Recruiters manage the flow of participants to the experiment."""
 
 import os
+import traceback
 
 from boto.mturk.connection import MTurkConnection
 from psiturk.models import Participant
 from psiturk.psiturk_config import PsiturkConfig
 
+from dallinger import db
+session = db.session
```

Comment:
this does not need to happen here, but this style is common across the rest of Dallinger so I suggest keeping it here too. If you don't like this at the top of the page we could add the import to the same function where the commit occurs? Though, we'll need to put it back up here if we want to edit the db elsewhere in recruiters